mkosi/__init__.py

```diff
@@ -50,6 +50,7 @@
 from mkosi.pager import page
 from mkosi.partition import Partition, finalize_root, finalize_roothash
 from mkosi.qemu import QemuDeviceNode, copy_ephemeral, run_qemu, run_ssh
+from mkosi.burn import run_burn
```

Comment:
CI complains because this should be between `mkosi.architecture` and `mkosi.config` in the imports.